frappe/hooks.py

```diff
@@ -568,6 +568,7 @@
 	"Activity Log": 90,
 	"Route History": 90,
 	"OAuth Bearer Token": 30,
+	"API Request Log": 7,
```

Comment:
I think the time is too short, as it is for audit purposes, it should be at least 90 days.

Reply:
Had kept it low as there will be a LOT of entries here, will increase the default but it can anyway be configured in Log Settings later.